scripts/run-tests.sh

```diff
@@ -1,21 +0,0 @@
-#!/bin/sh
```

Comment:
why have you removed this?

Reply:
Not needed, this can be replaced by a one liner (as specified in the README.md): `crystal tool format && bin/ameba && crystal spec`, then `shards build` can also be appended.